robot/subsystems/LineTracker.java

```diff
@@ -0,0 +1,57 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.AnalogInput;
+import edu.wpi.first.wpilibj.RobotDrive;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import frc.robot.DriveExecutor;
+import frc.robot.Robot;
+
+public class LineTracker extends Subsystem {
+
+  public static AnalogInput line1 = new AnalogInput(0);
+  private static AnalogInput line2 = new AnalogInput(1);
+  private static AnalogInput line3 = new AnalogInput(2);
+  public void lineInit() {
+AnalogInput.setGlobalSampleRate(62500);
+  }
+
+  int raw1  = line1.getValue();
+  int raw2 = line2.getValue();
+  int raw3 = line3.getValue();
+
+
+  public void lineFollowing() {
+    if(raw1 < 1000 && raw2 < 1000 && raw3 > 1000) {
+      Robot.driveExecutor.setY(0);
+    }
+    else if(raw1 < 1000) {
+      while(raw2 > 1000) {
```

Comment:
While loops are not the best to use with FRC. Scheduler calls it every 20ms. While adds overhead, use if.